lib/jekyll-admin/server/collection.rb

```diff
@@ -47,26 +49,70 @@ def collection
         collection[1] if collection
       end
 
+      def request_path
+        sanitized_path File.join(site.source, request_payload["path"])
```

Comment:
Thanks again 👍  it became very simple now https://github.com/jekyll/jekyll-admin/pull/281/commits/260ddaea8cdeedac1e908578606839150c9615a2